index.html

```diff
@@ -20,8 +20,8 @@
                         <div class="main-info">
                             <h1 id="main-h1">Monero</h1>
                             <h2 id="main-h2">Private Digital Currency</h2>
-                            <p id="main-text">Monero is a secure, private, untraceable currency. It is open-source and freely available to all. With Monero, you are your own bank. Only 
-                            you control and are responsible for your funds; your accounts 
+                            <p id="main-text">Monero is a secure, private and untraceable cryptocurrency. It is open-source and accessible to all. With Monero, you are your own bank. Only 
```

Comment:
I suggest we pull the updated verbiage out of #314 (_strings_en.yml). I don't like the almost accusatory language of "only you are responsible for your funds" and instead prefer instead suggesting that you have complete control. (Using positive language instead of negative.)
> Monero is a secure, private, untraceable currency that is open-source and freely available for anyone to use. With Monero, you are your own bank; You have complete control over your funds.